kea-vscode/src/views/common/local-commit/local-file-tree-node.test.ts

```diff
@@ -0,0 +1,186 @@
+import * as assert from "assert";
+import * as sinon from "sinon";
+import * as vscode from "vscode";
+import { IAccountKey } from "../../../account/account";
+import { IKeaContext } from "../../../core/context";
+import { ILocalGitRepository, LocalCommit } from "../../../git/local-git-repository";
+import { RepoId } from "../../../types/kea";
+import { createAccountStub, createKeaContextStub, createRepositoryStub } from "../../../test-utils";
+import { LocalFileTreeNode } from "./local-file-tree-node";
+
+suite("LocalFileTreeNode", () => {
```

Comment:
Please fix all the broken tests in this file. Make sure to compile the tests before running them.